include/mfmg/dealii/dealii_utils.hpp

```diff
@@ -79,6 +79,9 @@ void matrix_market_output_file(
     const std::string &filename,
     const dealii::TrilinosWrappers::SparseMatrix &matrix);
 
+void matrix_market_output_file(const std::string &filename,
+                               const dealii::SparseMatrix<double> &matrix);
```

Comment:
Please comment on how this gets called with the changes you propose?

Reply:
It does not, should have been a separate PR for that.

Reply:
Please remove from that PR.